src/main/java/uk/gov/di/ipv/cri/common/library/domain/personidentity/PersonIdentityDetailed.java

```diff
@@ -16,11 +16,18 @@ public class PersonIdentityDetailed {
     @JsonProperty("address")
     private final List<Address> addresses;
 
+    @JsonProperty("drivingPermit")
```

Comment:
Will this be ignored, when not supplied for non-driving licence cases

Reply:
It should be, the class has been marked as such:
`@JsonInclude(JsonInclude.Include.NON_NULL)`